test/e2e/specs/all-features-confirmation/navbar/no-logo.spec.js

```diff
@@ -1,4 +1,4 @@
-const { browser } = require('protractor');
+const { browser, element } = require('protractor');
```

Comment:
This doesn't actually need to be added since protractor attached browser/element to the global scope of the testing environment.

Reply:
I think I accidentally added `const { browser } = require('protractor');` and as you mentioned should be removed. Most probably when I was trying to use `browser` VS Code detected it and added this import statement. I'm assuming this is also the case for `element` that Ravish added.

Reply:
The same keeps happening to me as well